python/ingestion/merge_utils.py

```diff
@@ -162,8 +163,10 @@ def _merge_pop(df, demo, loc):
     pop_df = gcs_to_bq_util.load_df_from_bigquery(
         'acs_population', pop_table_name, pop_dtype)
 
-    needed_cols = [std_col.STATE_FIPS_COL, on_col_map[demo],
-                   std_col.POPULATION_COL, std_col.POPULATION_PCT_COL]
+    needed_cols = [on_col_map[demo], std_col.POPULATION_COL, std_col.POPULATION_PCT_COL]
+
+    if std_col.STATE_FIPS_COL in df.columns:
```

Comment:
do we need to send the state_fips for county-level files?

Reply:
It is in there for the cdc_restricted files for now. We can talk about that but I needed this in there for now.